pybloqs/static/__init__.py

```diff
@@ -1,78 +1,80 @@
-import itertools
-
-from pybloqs.html import js_elem, append_to, root, render
-from pybloqs.util import encode_string
-
 from pkg_resources import resource_filename
-
-
 from six import StringIO
 
+from pybloqs.html import js_elem, css_elem,  render
+from pybloqs.util import encode_string
+import os
+from io import open
```

Comment:
Small niggle, [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports) says stdlib imports should go first:

> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.